Npgsql.sln

```diff
@@ -55,6 +55,8 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Npgsql.DependencyInjection.
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Npgsql.NativeAotTests", "test\Npgsql.NativeAotTests\Npgsql.NativeAotTests.csproj", "{20F2E9D6-A69E-4BAE-9236-574B0AA59139}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "MStatDumper", "test\MStatDumper\MStatDumper.csproj", "{24F7AE57-EDB2-4932-AD5E-F2CE3ED59A3F}"
```

Comment:
Not sure it's worth including this in our solution (and having it loaded all the time) - we'll typically touch this very rarely (and can open that specific project what we do).